### ISSUE TITLE
Re-format the main class gist

### ISSUE THREAD

**Issue by mcantrelldev** [barrycumbie/jobie-19#29]
Clean up the class gist and add new Headings, Subheadings, Concise Bullet points to make the content more legible. 

Add collapse-able categories like Zoomland to each day to keep each day organized in its own space.

**Comment by barrycumbie** [comment]
@mcantrelldev "git the gist" 

Org overall wiki structure <> org a single page (template).
Create one nice, pretty, example page.

**Comment by mcantrelldev** [comment]
Some ideas - combine #30 #39 into this issue as they are related.

**Comment by gaplant** [comment]
I agree, I think they're close enough related to knock those two out. I'll go ahead and close #30.

**Comment by jeffreymccormick8** [comment]
@mcantrelldev 
Are you talking specifically about creating maybe a TOC that links to each day? Like in this image?
![Capture](https://user-images.githubusercontent.com/54875261/92151848-67079580-ede7-11ea-84f6-8ecf66055bc4.PNG)

**Comment by mzimmerman0** [comment]
I was thinking about maybe providing some sort of summary at the beginning of each day (topics covered during that day). That way we do not have to look through the entire day to see if one piece of info is there.

**Comment by mcantrelldev** [comment]
@jeffreymccormick8 Yes thats a good idea. A TOC at the top and have all days hidden and collapsible for faster scrolling when you only need to see a certain day.

**Comment by gaplant** [comment]
That's pretty much what I had in mind personally @jeffreymccormick8, https://gist.github.com/citrusui/07978f14b11adada364ff901e27c7f61 has some good examples for what we can do. It seems to handshake pretty well with Markdown so we can keep running with that.

**Comment by jeffreymccormick8** [comment]
I'll assign myself and see what I can come up with. I haven't quite figured out a way to get the links to actually open the drop-down.  Which would make it functional.

@mcantrelldev

**Comment by mcantrelldev** [comment]
> I'll assign myself and see what I can come up with. I haven't quite figured out a way to get the links to actually open the drop-down. Which would make it functional.
> 
> @jeffreymccormick8 


That would be neat if that functionality is possible in markdown. I was meaning just link right to the dropdown not open it but that would be really neat.

**Comment by mcantrelldev** [comment]
> I was thinking about maybe providing some sort of summary at the beginning of each day (topics covered during that day). That way we do not have to look through the entire day to see if one piece of info is there.
@mzimmerman0 

I think that is what Cumbie is aiming for already but yes we could do some edits to the daily summaries as they are kinda all over the place.

**Comment by jeffreymccormick8** [comment]
@mcantrelldev we certainly can do a link straight to the section.

**Comment by mcantrelldev** [comment]
> @mcantrelldev we certainly can do a link straight to the section.

@jeffreymccormick8 Sweet. That'll be a nice quality of life addition.

**Comment by jeffreymccormick8** [comment]
Where/how do we make this change?

**Comment by jeffreymccormick8** [comment]
> Where/how do we make this change?

@barrycumbie

**Comment by mcantrelldev** [comment]
> > Where/how do we make this change?
> 
> @barrycumbie
@jeffreymccormick8 

Great question. How do we actually edit the main gist? Do we submit our own markdown files to you and you apply changes like a shared excel sheet or can we get contributor access?

**Comment by desya42** [comment]
something kinda like this?
![gist](https://user-images.githubusercontent.com/59778932/92154344-45101200-edeb-11ea-9155-dba7cdbb1650.PNG)

**Comment by lovejoyy** [comment]
> something kinda like this?
> ![gist](https://user-images.githubusercontent.com/59778932/92154344-45101200-edeb-11ea-9155-dba7cdbb1650.PNG)

yes, perfect!

**Comment by desya42** [comment]
yea just not sure how to update the main gist i can only update my fork

**Comment by jeffreymccormick8** [comment]
@mcantrelldev 

From what Cumbie said in class, he mentioned trying to do separate wiki pages for days and topics. If that makes sense.

**Comment by mcantrelldev** [comment]
> @mcantrelldev
> 
> From what Cumbie said in class, he mentioned trying to do separate wiki pages for days and topics. If that makes sense.

@jeffreymccormick8 We could go that route aswell. Have tiny summaries on the gist but get more details with what was discussed on a given day on a seperate wiki page. We are going on the route for topics on wiki pages right now in tune with Alyssa's Issue #28

**Comment by mzimmerman0** [comment]
One thing that I can do is skim through the videos and time stamp the topics covered on each day

**Comment by jeffreymccormick8** [comment]
Anybody make a decision on what we need to do, or do I need to make that?

**Comment by mcantrelldev** [comment]
## GIST TO WIKI TEST
*Version 1.0: 2020-09-14 @mcantrelldev  *

### Instructions
- Do not use the class gist. Stick to the "Wiki" tab.
- Run through the test, attempting to continue past any invalid or failed steps, if at all possible.
- For all "No" reply and @mcantrelldev to say which step failed, why, and mention the exact step number.
- Test needs work / isn't clear @mcantrelldev and type ABANDONED, and mention which step you got to.
- Test successful - @mcantrelldev and type SUCCESSFUL, and include your answers to each test question.
- Record ideas for new features as a new issue. Don't forget to refer to this issue # as the origin of the idea.

### Test Script 
1. [ ] Can you find the daily class summaries? Y/N
1. [ ] Can you effectively learn how to do a pull request to our repo? Y/N
1. [ ] Can you find the class recordings? Y/N
1. [ ] Can you find out more about a sprint/scrum? Y/N
1. [ ] Can you find links to stuff like the markdown cheat sheet and where to install GIT for windows? Y/N
1. [ ] If you are unsure about a term can you look it up in the wiki effectively before googling? Y/N
1. [ ] Can you find how to create a issue in the same format as everyone else? Y/N

**Comment by barrycumbie** [comment]
@gaplant Topic: Cumbie's CIS 486 
Date: Sep 10, 2020 12:25 PM Central Time (US and Canada)

Meeting Recording:
https://una.zoom.us/rec/share/YECjkPa8_JPoB1WZ5LlWQP91W9-2PYttan5H_3lHKqmXrK9jPn5iZVXRGhQaxcLj.Z6etKz8aUgmW5cuh

Access Passcode: $crumb1e

**Comment by barrycumbie** [comment]
@gaplant 

Topic: Cumbie's CIS 486 
Date: Sep 15, 2020 12:25 PM Central Time (US and Canada)

Meeting Recording:
https://una.zoom.us/rec/share/sczqQ64MqurQ98ZTpQch5lkVlazpLHMn1DHZd6rA7Bp2HTAcg6I7S_cecPEMobaJ.mg-Thz0GlQInoFcH

Access Passcode: $crumb1e

**Comment by mattp281** [comment]
Tech Talk updated to Glossary in WIKI for more transparency.  Also WIKI Tools updated to show Issue #52 "how to add a milestone" per @barrycumbie .

**Comment by barrycumbie** [comment]
how to store quick and dirty info w/o doing a formal process?
